xbmc/platform/android/media/decoderfilter/MediaCodecDecoderFilterManager.cpp

```diff
@@ -39,10 +39,11 @@ CMediaCodecDecoderFilterManager::CMediaCodecDecoderFilterManager()
     NULL
   };
 
-  int num_codecs = CJNIMediaCodecList::getCodecCount();
-  for (int i = 0; i < num_codecs; i++)
+  std::vector<CJNIMediaCodecInfo> codecInfos =
+      CJNIMediaCodecList(CJNIMediaCodecList::REGULAR_CODECS).getCodecInfos();
+
+  for (auto codec_info : codecInfos)
```

Comment:
if it is not an iterator, please specify the type instead use `auto`
and probably worth not making copies of each object so add `&`

Reply:
and all others